models/topics-model.js

```diff
@@ -0,0 +1,19 @@
+const connection = require('../db/connection');
+const { topicData } = require('../db/data/test-data');
+
+const selectTopics = () => {
+    return connection.query(`SELECT * FROM topics`)
+                    .then(({rows : topics}) => {
```

Comment:
What do we mean by rows : topics here? I also noticed you have an OR operator to return an empty array but PSQL would return that anyway if it couldn't find any records